examples/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "examples",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "node-fetch": "^3.1.0"
+  }
+}
```

Comment:
If you're intending on forcing node to use ES6 module syntax, I suggest adding in the
```json
"type": "module"
```
attribute to the manifest as it'll force node to work in ES6 module format instead of CommonJS.
Ref: <https://stackoverflow.com/a/57492606>